### Footstep parameterization for a large humanoid, similar to ASIMO or HRP-2 ##

# cost to perform one footstep (in addition to distance costs, corresponds to 'meters')
step_cost: 0.1
# cost to estimate the difference between the current orientation and the goal 
# orientation; used in the heuristic it can easily overestimate the true path 
# cost since this cost is not incorporated in each step
diff_angle_cost: 0.0

### footstep settings ##########################################################
# Footstep displacement vectors (in meter / rad)
# WARNING: data type has to be float/double, integers will cause an exception
footsteps/x:     [0.0, 0.22, 0.00,-0.03, 0.14, 0.12, 0.08,-0.02,-0.10, 0.00]
footsteps/y:     [0.0, 0.00, 0.12, 0.00, 0.08,-0.02, 0.08, 0.08, 0.00,-0.02]
footsteps/theta: [0.0, 0.00, 0.00, 0.70, 0.30,-0.40, 0.00, 0.30, 0.00, 0.00]

### foot settings ##############################################################
foot:
  size: {x: 0.16, y: 0.10, z: 0.015}
  separation: 0.14
  origin_shift: {x: 0.02, y: 0.0}
  max:
    step: {x: 0.22, y: 0.12, theta: 0.7}
    inverse:
      step: {x: 0.1, y: 0.02, theta: 0.4}
accuracy:    
  # the execution accuracy of each footstep; the more accurate a footstep should
  # fit into the calculated steps the more likely a replanning is necessary
  footstep/x: 0.01
  footstep/y: 0.01
  footstep/theta: 0.15
